### PR TITLE
fix: fox assets names and icon overrides

### PR DESCRIPTION
**Pull request by firebomb1**
## Description
FOX assets on different networks in the app do not have the same icon and naming scheme. Let's fix this.

-  Overrides the name, icon and color for the 3 networks FOX is currently supported on in the app (Ethereum, Gnosis and Polygon)
- Updates the generatedAssets
- Updates the colorMap for assets

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #

## Risk
Low, but a lot of Cosmos pools seem to have been removed from Coingecko, might be worth investigating if any of these are in use by our users, but I'm not sure how to proceed here.

## Testing
Search for FOX in the Global Search or Assets page and notice all the ShapeShift FOX tokens use the same icons and same naming scheme "Fox on <network name>".

### Engineering
☝️ 

### Operations
☝️ 

## Screenshots (if applicable)
Before:
image

After:
image

### CODE REVIEW

**Comment by firebomb1** [comment]
Product reviewed [here](https://discord.com/channels/554694662431178782/886041582422356018/1114333940908576849).

Remains taking a look at the Cosmos assets that Coingecko seems to have dropped since last update, any idea if it's normal?

**Review by woodenfurniture — CHANGES_REQUESTED** [review]
Nice one @firebomb1 appreciate this! Looks like we shouldn't ditch those osmosis assets – the ones I've looked at are up and running, but there's a bit of a snag when it comes to how production is link to these assets in mintscan which means it's kinda a pain to check them without having to google each one. The good news is there's some patterns which means you might be able to sort it all out at once by tweaking the asset generator.

The osmosis pools can be found here
https://app.osmosis.zone/pools?pool=stable&pool=weighted&incentive=internal&incentive=external&incentive=superfluid&incentive=noIncentives

Here are the examples I checked so far:

## WBTC on osmosis

showing current market activity in prod:
https://app.shapeshift.com/#/assets/cosmos:osmosis-1/ibc:D1542AA8762DB13087D8364F3EA6509FD6F009A34F00426AF9E4F9FA85CBBF1F
image

currently listed as an active asset in mintscan:
https://www.mintscan.io/osmosis/assets/RDE1NDJBQTg3NjJEQjEzMDg3RDgzNjRGM0VBNjUwOUZENkYwMDlBMzRGMDA0MjZBRjlFNEY5RkE4NUNCQkYxRg==?type=ibc
image

however, prod is linking to the wrong page for the osmo asset - its linking to the account rather than the asset tracker:
https://www.mintscan.io/osmosis/account/D1542AA8762DB13087D8364F3EA6509FD6F009A34F00426AF9E4F9FA85CBBF1F
image

## USD Coin on Osmosis

showing current market activity in prod:
https://app.shapeshift.com/#/assets/cosmos:osmosis-1/ibc:D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858
image

currently listed as an active asset in mintscan, but has a different asset name to our app ( axlUSDC ???? ):
image

prod is linking to the wrong page for the osmo asset - its linking to the account rather than the asset tracker:
https://www.mintscan.io/osmosis/account/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858
image

## Osmosis OSMO/ATOM LP Token (gamm/pool/1)

Seems active in app, its liked to one of our defi staking pages:
https://app.shapeshift.com/#/assets/cosmos:osmosis-1/ibc:gamm/pool/1
image

Our app links to osmosis homepage, which is a problem:
https://www.mintscan.io/osmosis

Its available here though as their largest staking pool
https://app.osmosis.zone/pool/1
image

**Comment by firebomb1** [comment]
As in #4643 I've removed the changes that are not related to FOX assets. The asset-data generation script will have to be updated later in a separate PR, but at least we can get these changes in and they should remain when the script will be fixed for Cosmos/Osmosis assets.

File: scripts/generateAssetData/overrides.ts [hunk 1]
```diff
@@ -2,11 +2,23 @@ import type { AssetId } from '@shapeshiftoss/caip'
...
+  color: '#3761F9',
+  icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+}
+
 export const overrideAssets: Record<AssetId, Partial<Asset>> = {
   'eip155:1/erc20:0xc770eefad204b5180df6a14ee197d99d808ee52d': {
-    name: 'Fox',
-    color: '#3761F9',
-    icon: 'https://assets.coincap.io/assets/icons/256/fox.png',
+    name: 'Fox on Ethereum',
```
Comment (gomesalexandre) [inline scripts/generateAssetData/overrides.ts:12]: Do we want to use this PR to make it `FOX`?

**Review by gomesalexandre — APPROVED** [review]
Tested locally, LGTM

image

One pending question WRT Fox vs. FOX but obviously doesn't have to go in this PR

re: rugged Osmosis LP assets: https://github.com/shapeshift/web/pull/4650 should make this a lot better in the future - not sure whether that'll improve things as far as *Coingecko* assets go however

File: scripts/generateAssetData/overrides.ts (hunk 1 above)
Comment (gomesalexandre) [reply scripts/generateAssetData/overrides.ts:12]: Resolving so we can merge this regardless @firebomb1

**Pull request merged**